ansible/roles/ntp/defaults/main.yml

```diff
@@ -131,6 +133,7 @@ ntp__base_packages:
   - '{{ "chrony" if (ntp__daemon == "chrony") else [] }}'
   - '{{ "ntp" if (ntp__daemon == "ntpd") else [] }}'
   - '{{ "openntpd" if (ntp__daemon == "openntpd") else [] }}'
+  - '{{ "systemd-timesyncd" if (ntp__daemon == "systemd-timesyncd") else [] }}'
```

Comment:
Installing **systemd-timesyncd** might cause some issue on Debian Buster because it comes with systemd 241 by default.

systemd-timesyncd is only available in buster-backports and require to upgrade systemd to 245.4 (buster-backports).

Reply:
Buster and Stretch both have `systemd-timesyncd`...it's just that it's not in its own package, it's included in the [systemd package](https://packages.debian.org/buster/amd64/systemd/filelist), but yeah, good point, I can make the line above conditional on distro != buster/stretch.

Reply:
Done in a separate commit....